src/server/internalServer_catalog_v2.cpp

```diff
@@ -107,4 +109,16 @@ std::unique_ptr<Response> InternalServer::handle_catalog_v2_categories(const Req
   );
 }
 
+std::unique_ptr<Response> InternalServer::handle_catalog_v2_languages(const RequestContext& request)
+{
+  OPDSDumper opdsDumper(mp_library);
+  opdsDumper.setRootLocation(m_root);
+  opdsDumper.setLibraryId(m_library_id);
+  return ContentResponse::build(
+             *this,
+             opdsDumper.languagesOPDSFeed(mp_library->getBooksLanguages()),
```

Comment:
Do we have to pass the list of language as an argument as opdsDumper already knows the library ?
Couldn't it generate the languages list itself ?

(It is probably the same thing for the books categories)